docs/design/dlrover-overview.md

```diff
@@ -176,6 +176,48 @@ parameter servers to the Elastic Agent of all Pods. Then the Elastic
 Agent will notify the training framework (e.g. TensorFlow) to restart
 training and restore model paremeters from a checkpoint.
 
+
+#### Elasticity of AllReduce Training
```

Comment:
seems the background part above only covers ps training now,  should also be updated.